core/includes/form.inc

```diff
@@ -3037,6 +3038,7 @@ function form_process_password_confirm($element) {
     '#password_strength' => FALSE,
     '#password_shown' => FALSE,
     '#password_toggle' => FALSE,
+    '#spellcheck' => 'false',
```

Comment:
Is this line required? `theme_password()` does not use `$element['#spellcheck'], but set directly `$element['#attributes']['spellcheck'] to `'false'`.